### ISSUE TITLE
MAP - Menu - Profile Button

### ISSUE THREAD

**Issue by pjorritsma** [WatWowMap/ReactMap#369]
With the new features like Gym Badges would not be great to have a Menu Button on the ribbon for Profile?

Profile button could have the following features:

1.  Login / Logout
2.  Gym Badges
3.  Access
4. Import and Export 

So users don't need to go to Ribbon > Open Menu > Settings > Profile

**Comment by kamieniarz** [comment]
I agree, PR #372 is a great start for it. `Profile Settings` tab with all these things you pointed out could be added there

**Comment by clburlison** [comment]
This one is 90% completed with the merge of 372 as Kamieniarz mentioned. I don't think it makes sense to duplicate or move the Import/Export options into the Profile. Having those as part of Settings is two click away for users vs three clicks if it was moved under the Profile.

Feel free to reopen if you feel strongly about the Import/Export being moved/duplicated.
